src/components/request-content-modal/dialogs/text-request.tsx

```diff
@@ -0,0 +1,87 @@
+import { TypographyScale, Button, TextArea, TextInput } from '@mdb/flora';
+
+import { TextRequestProps } from '../types';
+import { modalWrapperStyles } from '../styles';
+import { useState } from 'react';
+
+const helperText =
+    "Please include which products, languages, and technologies you're interested in seeing as part of this content request";
+
+const TextRequest: React.FunctionComponent<TextRequestProps> = ({
+    onContinue,
+    contentCategory,
+}) => {
+    const title = `Request ${
+        /^[aeiou]/gi.test(contentCategory) ? 'an' : 'a'
+    } ${contentCategory}`; // Regex to tell if it starts with a vowel.
+
+    const [topic, setTopic] = useState('');
+    const [description, setDescription] = useState('');
+    const [email, setEmail] = useState('');
+
+    return (
+        <div sx={modalWrapperStyles}>
+            <TypographyScale
+                variant="heading5"
+                sx={{
+                    display: 'block',
+                    marginBottom: ['inc40', null, null, 'inc50'],
+                }}
+            >
+                {title}
+            </TypographyScale>
+
+            <div
+                sx={{
+                    '& > div': {
+                        width: '100%',
+                    },
+                    marginBottom: 'inc30',
+                }}
+            >
+                <TextInput
+                    name="topic"
+                    label="Topic"
+                    value={topic}
+                    onChange={e => setTopic(e.target.value)}
+                />
+            </div>
+
+            <div sx={{ marginBottom: 'inc30' }}>
+                <TextArea
+                    name="description"
+                    label="Describe your experience"
+                    value={description}
+                    onChange={e => setDescription(e.target.value)}
```

Comment:
So I've been reading about the performance of callbacks like this, and [this article](https://dmitripavlutin.com/dont-overuse-react-usecallback/) kinda made me think in terms of "if it becomes a problem we'll fix it", rather than overcomplicating the code. Basically we'd have to write a `useCallback` hook for each field, and I'm not super convinced that avoiding defining a handful of functions is worth the complexity.

I just profiled it and it really doesn't look like much (<1ms).